src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -586,6 +600,170 @@ def image_mask_iterator():
     return nwbfile
 
 
+def add_fluorescence_traces(
+    segmentation_extractor: SegmentationExtractor,
+    nwbfile: NWBFile,
+    metadata: Optional[dict],
+    plane_index: int = 0,
```

Comment:
This is called `plane_segmentation_index` in the other functions. I also realized that in the top level function `write_segmentation` we have yet another name called `plane_no` (!).

Reply:
I'll check,

Reply:
Leave this as it is, as I think is too big already. I discussed this with @CodyCBakerPhD today and I will take care of this separately after this PR.